src/Microsoft.Diagnostics.Runtime/Native/NativeRuntime.cs

```diff
@@ -261,6 +275,95 @@ private void InitThreads()
             _threads = threads.ToArray();
         }
 
+        public override IEnumerable<ClrException> GetSerializedExceptions()
+        {
+            if (!this.ClrInfo.DacInfo.PlatformAgnosticFileName.Contains("mrt100"))
```

Comment:
You should remove this if statement.  The only way to construct this class (NativeRuntime) is if we are debugging a .Net Native process.  This is also the wrong way to check for .Net Native...but since this check is completely not needed, you should just remove this if statement.